k8s/eks/yaml/tg-ds-sidecar.yml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: testground-sidecar
+spec:
+  selector:
+    matchLabels:
+      name: testground-sidecar
+  template:
+    metadata:
+      labels:
+        name: testground-sidecar
+    spec:
+      terminationGracePeriodSeconds: 10
+      hostPID: true
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      priorityClassName: system-node-critical
+      containers:
+      - name: testground-sidecar
+        image: public.ecr.aws/n6b0k8i7/testground-sidecar:eks_v1.0
+        imagePullPolicy: Always
+        command: ["testground"]
+        args: ["sidecar", "--runner", "k8s"]
+        securityContext:
+          capabilities:
+            add: ["NET_ADMIN", "SYS_ADMIN", "SYS_TIME"]
+          privileged: true
+        env:
+        - name: LOG_LEVEL
+          value: "DEBUG"
+        ports:
+        - name: sidecarhttp
+          containerPort: 6060
+        resources:
+          limits:
+            memory: 900Mi
```

Comment:
Explained the increase [here](https://github.com/testground/testground/issues/1454).

Reply:
Same - https://github.com/testground/infra/pull/78#discussion_r984346879

Reply:
Added a comment in the yaml.